autogen/agentchat/groupchat.py

```diff
@@ -633,12 +645,17 @@ def validate_speaker_name(recipient, messages, sender, config) -> Tuple[bool, Un
             human_input_mode="NEVER",  # Suppresses some extra terminal outputs, outputs will be handled by select_speaker_auto_verbose
         )
 
+        # Add the message transforms, if any, to the speaker selection agent
+        if self._auto_message_transforms is not None:
+            self._auto_message_transforms.add_to_agent(speaker_selection_agent)
+
         # Run the speaker selection chat
         result = checking_agent.initiate_chat(
             speaker_selection_agent,
             cache=None,  # don't use caching for the speaker selection chat
             message={
                 "content": self.select_speaker_prompt(agents),
+                "name": "checking_agent",
```

Comment:
Maybe this is line change should be part of #2635 ?

Reply:
Yes, I will put these in #2635

Reply:
Okay, all done on #2635 side. I'll remove from here and commit.